src/views/user/index.js

```diff
@@ -140,7 +146,11 @@ class UserView extends React.Component<Props, State> {
             title={title}
             description={description}
             image={user.profilePhoto}
-          />
+            type="profile"
+          >
+            <meta property="profile:last_name" content={user.name} />
```

Comment:
Why do you we want a last_name property that won't be accurate?

Reply:
Because we only have firstname and full name, so I wasn't sure what to do here. (Open Graph only supports `first_name` and `last_name`) I figured this was better than duplicating the first name?